docs/management/api/host-isolation-exceptions-api.asciidoc

```diff
@@ -0,0 +1,469 @@
+:endpoint-artifact-list-id:                     endpoint_host_isolation_exceptions
+:endpoint-artifact-name:                        host isolation exceptions
+:endpoint-artifact-name-single:                 host isolation exception
+:endpoint-artifact-name-sentence-start:         Host isolation exceptions
+:endpoint-artifact-name-single-sentence-start:  Host isolation exception
+[[host-isolation-exceptions-api]]
+=== {endpoint-artifact-name-sentence-start}
+
+Create, retrieve, update, and delete endpoint <<host-isolation-exceptions>> via API. {endpoint-artifact-name} are managed via the <<exceptions-api-overview>> using a static container id (`list_id`) of `pass:a[{endpoint-artifact-list-id}]`, which must be created prior to adding a trusted application. Access to these APIs requires that a user has authorization to manage endpoints.
```

Comment:
Use the start of sentence attribute here + reference the attribute for artifact type:

```suggestion
Create, retrieve, update, and delete endpoint <<host-isolation-exceptions>> via API. {endpoint-artifact-name-sentence-start} are managed via the <<exceptions-api-overview>> using a static container id (`list_id`) of `pass:a[{endpoint-artifact-list-id}]`, which must be created prior to adding the {endpoint-artifact-name}. Access to these APIs requires that a user has authorization to manage endpoints.
```